jobserver/views/jobs.py

```diff
@@ -59,23 +59,31 @@ def get(self, request, *args, **kwargs):
         # reliably see the first and last honeycomb events relating to that job.
         # This could be due to clock skew, '>' rather than '>=' comparators
         # and/or other factors.
-        honeycomb_context_starttime = job.created_at - timedelta(minutes=1)
+        honeycomb_context_starttime = int(
+            (job.created_at - timedelta(minutes=1)).timestamp()
+        )
 
         honeycomb_context_endtime = timezone.now()
         if job.completed_at is not None:
             honeycomb_context_endtime = job.completed_at + timedelta(minutes=1)
 
+        honeycomb_context_endtime_unix = int(honeycomb_context_endtime.timestamp())
+
         context = {
             "job": job,
             "object": job,
             "user_can_cancel_jobs": can_cancel_jobs,
             "view": self,
             "honeycomb_can_view_links": honeycomb_can_view_links,
-            "honeycomb_context_starttime": honeycomb_context_starttime,
-            "honeycomb_context_endtime": honeycomb_context_endtime,
+            "honeycomb_link": f"https://ui.honeycomb.io/bennett-institute-for-applied-data-science/environments/production/datasets/job-server?query=%7B%22start_time%22%3A{honeycomb_context_starttime}%2C%22end_time%22%3A{honeycomb_context_endtime_unix}%2C%22granularity%22%3A0%2C%22breakdowns%22%3A%5B%22status%22%5D%2C%22calculations%22%3A%5B%7B%22op%22%3A%22COUNT%22%7D%5D%2C%22filters%22%3A%5B%7B%22column%22%3A%22name%22%2C%22op%22%3A%22%3D%22%2C%22value%22%3A%22update_job%22%7D,%7B%22column%22%3A%22job_identifier%22%2C%22op%22%3A%22%3D%22%2C%22value%22%3A%22{job.identifier}%22%7D%5D%2C%22filter_combination%22%3A%22AND%22%2C%22orders%22%3A%5B%7B%22op%22%3A%22COUNT%22%2C%22order%22%3A%22descending%22%7D%5D%2C%22havings%22%3A%5B%5D%2C%22limit%22%3A100%7D",
```

Comment:
The previous link had a `default_if_none:'1646312416'` for the start time, is that still needed?

Reply:
@ghickman do you know the answer to this?

Reply:
Ah, yes, this is because there will never be an empty start time since we generate it from `created_at`, so we didn't need the default.